### PR TITLE
Added GetDebugInfo

### PR DESCRIPTION
**Pull request by cielavenir**
(note: https://github.com/mujin/controllerclientcpp/pull/74 and https://github.com/mujin/mujincontrollerclientpy/commits/applyconfig2 effort was done to maintain consistency the return type of this function.)

### CODE REVIEW

**Comment by ziyan** [comment]
It's `DebugResource` not `DebugInfo`.

File: python/mujincontrollerclient/controllerclientbase.py [hunk 1]
```diff
@@ -975,3 +975,19 @@ def SetITLProgram(self, programName, data, fields=None, usewebapi=True, timeout=
...
+        assert(usewebapi)
+        params = {
+            'offset': offset,
+            'limit': limit,
+        }
+        return self.ObjectsWrapper(self._webclient.APICall('GET', u'/debug/', timeout=timeout, params=params))
+
+    def GetDebugInfo(self, pk, usewebapi=True, timeout=60, **kwargs):
+        assert(usewebapi)
+        response = self._webclient.Request('GET', '/api/v1/debug/'+pk+'/download/', timeout=timeout, stream=True)
```
Comment (ziyan) [inline python/mujincontrollerclient/controllerclientbase.py:989]: There is a `downloadUri` that you should follow.

Comment (ziyan) [reply python/mujincontrollerclient/controllerclientbase.py:989]: This should be probably be `DownloadDebugResource`.

**Comment by ziyan** [comment]
By the way, why are you adding this to controllerclientpy? If it is not ABSOLUTELY necessary, I do not want to expose more endpoints.

**Comment by cielavenir** [comment]
We are trying to use module-ci more, which cannot add cpp tests.

/cc @kktk-KO

**Comment by ziyan** [comment]
Can you wait until webstack2 is merged?

**Comment by cielavenir** [comment]
no problem, this is Python and monkey-patch (in test code) is possible.

**Comment by cielavenir** [comment]
https://github.com/mujin/mujincontrollerclientpy/commit/55a0126c4b24609169ecb6c68d3dc07f7330f65d does the same thing, closing

/cc @woswos

**Pull request closed without merging**